app/src/main/res/values/strings.xml

```diff
@@ -564,4 +564,16 @@
     <string name="call_without_notification">Call without notification</string>
     <string name="set_avatar_from_camera">Set avatar from camera</string>
 
+    <string name="nc_conversation_settings">Conversation settings</string>
+
+    <!-- Expiring messages -->
+    <string name="nc_expire_messages">Expiring messages</string>
```

Comment:
```suggestion
    <string name="nc_expire_messages">Expire chat messages</string>
```

Also if possible can we copy the explanation from the web UI part, to explain what happens to shared files, etc?

Reply:
As discussed, I would like to see it, if it looks out of place lets skip it for now.
Not worth looking into a new lib atm.

Reply:
this would be the dirty solution with a normal text field under the setting:

![grafik](https://user-images.githubusercontent.com/2932790/186186496-d47f5695-e210-4170-ad82-64624279e214.png)

:+1:  or :-1:  @nickvergessen ?

Reply:
@mahibi the new header line theme to not be tinted correctly, seems to be a lighter shade of blue (best to test on sermo if it turns orange)

Reply:
I think that's good enough?